packages/ui-components/src/Common/Search/Suggestions/index.tsx

```diff
@@ -18,8 +18,12 @@ const SearchSuggestions: FC<SearchSuggestionsProps> = ({
   <Suggestions.Wrapper className={styles.suggestionsWrapper}>
     {label && <p className={styles.suggestionsTitle}>{label}</p>}
     {suggestions.map((suggestion, i) => (
-      <Suggestions.Item {...props} key={i} className={styles.suggestionItem}>
-        <SparklesIcon />
+      <Suggestions.Item
```

Comment:
I put a breakpoint within https://github.com/oramasearch/orama-ui/blob/main/packages/ui/src/components/Suggestions.tsx#L53-L59 - and I can see the stream return from the `useChat` `ask` call.